.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!-- Thank you for contributing a pull request! Here are a few tips to help you:
+
+1. If this is your first contribution, make sure you've read the Contribution Checklist https://developers.mattermost.com/contribute/getting-started/contribution-checklist/
+2. Read our blog post about "Submitting Great PRs" https://developers.mattermost.com/blog/2019-01-24-submitting-great-prs
+3. Take a look at other repository specific documentation at https://developers.mattermost.com/contribute
+-->
+
+#### Summary
+<!--
+A description of what this pull request does.
+-->
+
+#### Ticket Link
+<!--
+If this pull request addresses a Help Wanted ticket, please link the relevant GitHub issue, e.g.
+
+  Fixes https://github.com/mattermost/mattermost-server/issues/XXXXX
+
+Otherwise, link the Jira ticket.
+-->
+
+#### Checklist
```

Comment:
@hanzei, would you consider sharing this with the team at large? This particular checklist predates me, so I don't have as much context into perhaps what went into the thinking here.

Reply:
Sure, will share with in the Contributors channel.

Reply:
Done https://community-daily.mattermost.com/core/pl/1ikmqhzw1ffsfepzyb4575bkih